### PR TITLE
Wdi16

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -186,82 +237,244 @@ const results = [
...
+- The conditional rendering will be delegated to the `SearchContainer` component
+
+#### In `SearchContainer.js`...
+
+- Paste the conditional logic that controls whether the user sees `<Search />` or `<Results />` into the render method, making sure that its dependent on the `hasSearched` value in state (not props)
+- Update `<Search />` in the UI so that it takes in `this.state.query` via props as `query`
+- Update `<Results />` in the UI so that it takes in `this.state.shows` via props as `shows`
+
+## Catch-Up & Break (15 minutes / 2:05)
+
```
Comment (nayana487) [inline readme.md:391]: What's in this "catch up"? is this a review?

Saw that you have a break earlier too without a catch up. Might wanna keep consistent

File: readme.md [hunk 2]
```diff
@@ -186,82 +237,244 @@ const results = [
...
+}
+```
+
+When your done with this section, [your code should look something like this](https://github.com/ga-wdi-exercises/react-tvmaze/tree/07a579736dd40488dbab943c695e329ab61ae2ba).
+
+## Replace Mock Data with an AJAX Request (10 minutes / 2:30)
+
+Finally, we will replace the mock data with an actual AJAX request and update the `onSubmitQuery` to update state in handling the resolved promise.
 
 #### [AJAX request](https://github.com/ga-wdi-exercises/react-tvmaze/commit/8bb4f4edd5a98261b2a89116e40ca20e9f025269)
```
Comment (nayana487) [inline readme.md]: are they just copying the code in the link? trying to figure it out on their own? or are you showing them this? lil unclear

File: readme.md [hunk 3]
```diff
@@ -186,82 +237,244 @@ const results = [
...
+  // ...
+  render(){
+    const toRender = this.state.hasSearched
+      ? <Results shows={this.state.shows} />
+      : <Search query={this.state.query} onSubmitQuery={this.onSubmitQuery} />
+    return <div>{toRender}</div>
+  }
+}
+```
+
```
Comment (nayana487) [inline readme.md:438]: do they know about the ternary operator?

File: readme.md (hunk 1 above)
Comment (amaseda) [reply readme.md:391]: It's just some additional time if we need to help students troubleshoot after the "We Do" is finished.

File: readme.md (hunk 3 above)
Comment (amaseda) [reply readme.md:438]: I've used it in class a few times with them, but I'll show them different ways of implementing conditional logic here.

File: readme.md (hunk 2 above)
Comment (amaseda) [reply readme.md]: Killed this section.

**Pull request merged**